opentracing-spring-web-autoconfigure/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>io.opentracing.contrib</groupId>
     <artifactId>opentracing-spring-web-parent</artifactId>
-    <version>0.0.11-SNAPSHOT</version>
+    <version>0.1.0-RC1-SNAPSHOT</version>
```

Comment:
nit: less error-prone is to leave this for the release script.

Reply:
Understand - do you want me to revert or just leave for now?

Reply:
no it's fine, it just can be done automatically :)